roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -43,3 +43,4 @@
     baseurl: "http://koji.katello.org/releases/yum/katello-{{ katello_repositories_version }}/client/el{{ ansible_distribution_major_version }}/x86_64/"
     priority: 1
     gpgcheck: 0
+  when: katello_repositories_version != 'nightly' or katello_repositories_version < 3.9
```

Comment:
The only thing I wonder about is if the type is correct. What if it's a string instead of a float?